bindings/ergo-lib-c-core/Cargo.toml

```diff
@@ -14,6 +14,7 @@ crate-type = ["lib"]
 base16 = "0.2.1"
 derive_more = "0.99"
 ergo-lib = { version = "^0.14.0", path = "../../ergo-lib" }
+ergo-merkle-tree = { version = "0.1.0", path = "../../ergo-merkle-tree" }
```

Comment:
To keep `bindings/ergo-lib-*` crates dependencies minimal I'd rather have it as a re-export in `ergo-lib` along with other dependencies - https://github.com/ergoplatform/sigma-rust/blob/95e533beb41ce3667766a1a869e11cc6ff3e1850/ergo-lib/src/lib.rs#L27-L34
The same goes for `ergo-lib-wasm`.